docs/fastdds/use_cases/reduce_memory/reduce_memory.rst

```diff
@@ -0,0 +1,97 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+
+.. _use-case-reduceMemory:
+
+Reduce the usage of memory
+==========================
+
+A great number of modern systems have tight constraints over available memory. Reducing the memory usage is critical for
+these applications. Reducing memory consumption of a Fast-DDS application can be achieved through a different number of
+approaches, mainly through architectural restructuring, but also via the limiting of resources and by avoiding static
+allocations.
+
+If a system has a low memory limit it is recommended to limit the maximum resources of the application to ensure that it
+will run properly. To lower the usage during runtime, allocations can be set dynamically to ensure the lowest amount of
+memory is always in use.
```

Comment:
I think this is the same info as the previous paragraph with a different formulation